action/show.go

```diff
@@ -91,7 +92,7 @@ func (s *Action) show(c *cli.Context, name, key string, clip, force, qr bool) er
 		}
 	}
 
-	fmt.Println(color.YellowString(string(content)))
+	fmt.Println(color.YellowString(strings.Trim(string(content), "\r\n")))
```

Comment:
unsing `strings.TrimSpace` would be more idiomatic

Reply:
On reflection I reduced this to trimming trailing \n's or \r's in case leading whitespace is part of the password.